### PR TITLE
mounter.go: format return err.

### PR DESCRIPTION
**Pull request by zhangxiaoyu-zidif**
**What this PR does / why we need it**:
when an error returned is nil, it's preferred to explicitly return nil.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by grodrigues3** [comment]
@k8s-bot ok to test

File: cluster/gce/gci/mounter/mounter.go [hunk 1]
```diff
@@ -68,7 +68,7 @@ func mountInChroot(rootfsPath string, args []string) error {
 	args = append([]string{rootfsPath, mountCmd}, args...)
 	output, err := exec.Command(chrootCmd, args...).CombinedOutput()
 	if err == nil {
-		return err
+		return nil
```
Comment (k82cn) [inline cluster/gce/gci/mounter/mounter.go]: should be ?
```
if err != nil {
    return err
}
```

Comment (zhangxiaoyu-zidif) [inline cluster/gce/gci/mounter/mounter.go]: Thank you for reviewing. I think  "return nil' is correct. That means command executed successfully. So it has no error and returns.

Comment (k82cn) [inline cluster/gce/gci/mounter/mounter.go]: wow, just check the code; that's right :).

Comment (CaoShuFeng) [inline cluster/gce/gci/mounter/mounter.go]: Is there any insufficient of the original code?
I don't think the original code is wrong, so....

Comment (zhangxiaoyu-zidif) [inline cluster/gce/gci/mounter/mounter.go]: image

Comment (zhangxiaoyu-zidif) [inline cluster/gce/gci/mounter/mounter.go]: @CaoShuFeng  Not right or wrong, just proper or improper. Thx.

Comment (CaoShuFeng) [inline cluster/gce/gci/mounter/mounter.go]: Nice, I suggest to invate kargakis to review this code.

**Comment by gmarek** [comment]
@kargakis

**Comment by zhangxiaoyu-zidif** [comment]
@jszczepkowski  The PR needs your APPROVED

**Pull request merged**